06-files-input-and-output/remove_files.py

```diff
@@ -1,49 +0,0 @@
-# Copyright (c) 2016 Ami . All rights reserved
```

Comment:
there was a master conflict these files are not in master and accidently added so i fixed the conflict by merging master to this branch and removing that file